contracts/DLT/DLT.sol

```diff
@@ -379,14 +381,10 @@ contract DLT is Context, IDLT {
             ++_totalSubIds[mainId];
         }
 
-        unchecked {
-            _totalSupply += amount;
-            _mainTotalSupply[mainId] += amount;
-            _subTotalSupply[mainId][subId] += amount;
-
-            _balances[mainId][account].mainBalance += amount;
-            _balances[mainId][account].subBalances[subId] += amount;
-        }
+        _totalSupply += amount;
```

Comment:
should we keep the total supply? is it required here?

Reply:
no